README.md

```diff
@@ -512,35 +512,64 @@ Polluting globals is a bad practice in very languages because you could clash wi
 library and the user of your API would be none-the-wiser until they get an exception in 
 production. Let's think about an example: what if you wanted to have configuration array. 
 You could write global function like `config()`, but it could clash with another library 
-that tried to do the same thing. This is why it
-would be much better to use singleton design pattern and simple set configuration.
+that tried to do the same thing.
 
 **Bad:**
 ```php
-function config() {
+function config()
+{
     return  [
-        'foo': 'bar',
+        'foo' => 'bar',
     ]
 }
 ```
 
 **Good:**
+
+Create configuration file
+
 ```php
-class Configuration {
-    private static $instance;
-    private function __construct($configuration) {/* */}
-    public static function getInstance() {
-        if (self::$instance === null) {
-            self::$instance = new Configuration();
-        }
-        return self::$instance;
+// config.php
+return [
+    'foo' => 'bar',
+];
+```
+
+Or a [YAML](https://en.wikipedia.org/wiki/YAML) configuration file
```

Comment:
I'd keep it simple with 1 example to do things. That would keep point more clear.